test/jdk/com/sun/jdi/JdwpNetProps.java

```diff
@@ -67,19 +67,26 @@ public static void main(String[] args) throws Exception {
             new ListenTest("localhost", ipv4Address)
                     .preferIPv4Stack(false)
                     .run(TestResult.Success);
+            new ListenTest("localhost", ipv4Address)
+                    .preferIPv4Stack(true)
+                    .preferIPv6Addresses("system")
+                    .run(TestResult.Success);
             if (ipv6Address != null) {
-                // - only IPv4, so connection prom IPv6 should fail
+                // - only IPv4, so connection from IPv6 should fail
                 new ListenTest("localhost", ipv6Address)
                         .preferIPv4Stack(true)
-                        .preferIPv6Addresses(true)
+                        .preferIPv6Addresses("true")
                         .run(TestResult.AttachFailed);
                 // - listen on IPv4
                 new ListenTest("localhost", ipv6Address)
-                        .preferIPv6Addresses(false)
+                        .preferIPv6Addresses("false")
```

Comment:
It seems we should also have a test case that omits using `preferIPv6Addresses()`. I think like this test case, it should fail because the debug agent will default to v4.

Reply:
Done